roles/online_docs/templates/mkdocs/docs/apptainer.md

```diff
@@ -0,0 +1,595 @@
+- `%labels` are key-value pairs, and define the metadata of your container (f.e. `Author d@sylabs.io`),
+- `%help` at build time is added to the metadata of the container, and can be displayed using the run-help command,
+- `%post` section executes within the container at build time after the base OS has been installed. The `%post` section is therefore the place to perform installations of new applications.
+
+You can read more about the definition file on the [Apptainer's documentation website](https://docs.sylabs.io/guides/3.7/user-guide/definition_files.html)
+
+## Important
+
+It depends on how the container was built, but often `/home` folder is shared between the container and the host machine. **This means that the users home folder files can be changed or deleted just by running a container**. Make sure you check first (either by running with `shell` option) and then run an isolated environment (check the section about `Isolated runs`).
+
+Word of advice: docker and apptainer repositories are developed and maintained by the community, and therefore **images can and do contain all sorts of software and configurations**. While most of them are safe to use, users should be careful and avoid running containers from unknown sources. They might contain either buggy software or malicious code. Try to run only trusted images and when not sure, first try running inside an isolated environment (check the section about `Isolated runs`).
+
+Storage consumption: each container consumes a lot of storage. Making a new container will quickly fill up the disk. Make sure you keep only containers that you need. At the same time, pulling containers from public repositories, stores image layers locally in the `~/.apptainer/cache` folder. Users home folder can be quickly completely filled up with these temporary files. Caching shoud be disabled or redirected (see the section about `Caching redirect or disable`).
+
+## Caching redirect or disable
+
+By default the apptainer caches all the downloaded image layers. Simply pulling the docker image on Gearshift with
+
+```
+    $ apptainer pull gate-9.2.sif docker://opengatecollaboration/gate:9.2-docker
+    INFO:    Converting OCI blobs to SIF format
+    INFO:    Starting build...
+    Getting image source signatures
+    Copying blob c84400a81634 [=============================>--------] 213.6MiB / 273.7MiB
+    Copying blob bdf5ec5a2e5d done  
+    Copying blob ebbbc5f611f3 done  
+    Copying blob 1a930d163dca done  
+    Copying blob 1d58a538c5fc [=======>------------------------------] 219.8MiB / 1.0GiB
+    Copying blob 7da40ae7e7dd done  
+    ...
+```
+
+will fail, as the layer `blobs` will be by default downloaded inside your home's `~/.apptainer/cache` directory. It will fill it up until 1.3GB and then the system quota will prevent further writing of the data - making the apptainer command fail.
```

Comment:
```suggestion
will fail when the default `~/.apptainer/cache` directory is used. It will fill it up until the quota limit for your home directory is reached, which will prevent any further writes making the `apptainer` command fail.
```